src/messenger-shortcuts.js

```diff
@@ -78,6 +92,16 @@ function last(arr) {
   return arr.length === 0 ? undefined : arr[arr.length - 1];
 }
 
+function getCode(index) {
+  return bind[index].charCodeAt(0);
+}
+
+function dismiss() {
+	if(menuOpened) {
```

Comment:
Also, I don't think this function actually does anything. Clicking _already_ closes the help dialog. This is because the help dialog is just the mute dialog with its contents replaced by `HELP_HTML`. I'm basically hijacking the Messenger UI. 😉